### PR TITLE
docs: Use apidoc directive in typed/persistence.md - the last part (#22904)

### PR DESCRIPTION
**Pull request by Captain1653**
References #22904 

It's the last part of typed/persistence.md

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/persistence.md [hunk 1]
```diff
@@ -602,20 +602,20 @@ Scala
...
 Journals can reject events. The difference from a failure is that the journal must decide to reject an event before
 trying to persist it e.g. because of a serialization exception. If an event is rejected it definitely won't be in the journal. 
-This is signalled to an `EventSourcedBehavior` via an `EventRejectedException` and can be handled with a @ref[supervisor](fault-tolerance.md).
+This is signalled to an @apidoc[typed.*.EventSourcedBehavior] via an @apidoc[typed.EventRejectedException] and can be handled with a @ref[supervisor](fault-tolerance.md).
 Not all journal implementations use rejections and treat these kind of problems also as journal failures. 
 
 ## Stash
 
-When persisting events with `persist` or `persistAll` it is guaranteed that the `EventSourcedBehavior` will not receive
+When persisting events with `persist` or `persistAll` it is guaranteed that the @apidoc[typed.*.EventSourcedBehavior] will not receive
```
Comment (Captain1653) [inline akka-docs/src/main/paradox/typed/persistence.md:618]: I'm not sure about links `persist` and `persistAll`.

**Comment by Captain1653** [comment]
Looks like some flacky test https://github.com/akka/akka/runs/7983163543?check_suite_focus=true#step:5:1413

**Comment by He-Pin** [comment]
@Captain1653 that issue have been fixed, tirgger a rerun?

**Review by johanandren — APPROVED** [review]

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by Captain1653** [comment]
@patriknw  2.6.20 milestone?

**Pull request merged**